dali/kernels/imgproc/convolution/cutlass/device/gemm.h

```diff
@@ -469,8 +469,9 @@ class Conv {
     }
 
     size_t params_sizeof = host_params_.size() * sizeof(SampleParams);
-    cudaMemcpyAsync(params_.params, host_params_.data(), params_sizeof, cudaMemcpyHostToDevice,
-                    stream);
+    CUDA_CALL(
```

Comment:
This is actually wrong - the status is checked and the function is apparently not supposed to throw.

Reply:
done